src/components/OrganizationAccordion.tsx

```diff
@@ -47,7 +48,13 @@ export const OrganizationAccordion = ({
     return null;
   }
 
-  const expanded = isExpanded || (!!searchId && !includeAllSubunits);
+  let expanded = isExpanded || (!!searchId && !includeAllSubunits);
+
+  // Close all expanded when searches are reset
+  if (!searchId && !selectedId) {
```

Comment:
Added this code so that when we reset the subunit-field, the previously selected value stops being expanded.

Reply:
Ser samme gjelder andre plasser også ja!
Kanskje slå sammen disse til noe sånt da?
```
const isSearchedUnit = !!searchId && !includeAllSubunits; // Har ikke noe godt forslag til navn her dessverre
const hasResetSearch = !searchId && !selectedId;
 const expanded = (isExpanded || isSearchedUnit) && !hasResetSearch;
```
Virker så vilkårlig nå at det er en rar sjekk over som vi begge sliter med å forstå, så etterpå kommer det en if som kanskje overstyrer den uansett 😅

Reply:
Ja jeg ville egentlig ha en kommentar over for å forklare hva den gjør siden det er ganske vanskelig å skjønne. Men jeg skjønte ikke helt forklaringen din nå heller, så foretrekker å ikke trekke dem sammen for å gjøre det enda mer komplisert (akkurat nå forklares i det minste deler av logikken)